core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/footer_de.properties

```diff
@@ -22,3 +22,4 @@
 
 Manage\ Jenkins=Jenkins verwalten
 tooltip={0,choice,0#Keine Administrator-Warnungen sind|1#{0} Administrator-Warnung ist|1<{0} Administrator-Warnungen sind} aktiv.
+monitors={0,choice,0#Warnungen|1#Warnung|1<Warnungen}
```

Comment:
@uhafner do you think this one's right? I tried to set the German translation based on the previous line (didn't try with the other translations for this file)

Reply:
Yes that is correct!